more_autograding_examples/networking/config/test_output/google.txt

```diff
@@ -0,0 +1 @@
+if (!iesg){document.f&&document.f.q.focus();document.gbqf&&document.gbqf.q.focus();}
```

Comment:
It might make more sense to wget a file we control (potentially from https://submitty.org) to avoid further edits due to google changing their site.

Reply:
@bmcutler thoughts on making some kind of an "unchanging" page on submitty.org? I'm worried about putting an open-source page as part of the testcase, so I'm not sure how to approach which page would be best to use.